scripts/extract_translation_templates.py

```diff
@@ -90,11 +91,36 @@ def check_schema_templates(source_dir, target_dir):
 
 
 if __name__ == "__main__":
+
+    response = requests.get("https://api.github.com/repos/ONSdigital/eq-translations/commits/master").json()
```

Comment:
This should probably be in its own script

Reply:
Also, if we do start using releases of eq-translations it will greatly simplify the check, as we can use pipenv --dry-run to check for an update.

Reply:
Moved it to it's own script and using version of eq-translations package now, thanks👍